src/overlays/actors/ovl_En_Elforg/z_en_elforg.c

```diff
@@ -70,34 +69,37 @@ void EnElforg_Init(Actor* thisx, PlayState* play) {
     s32 pad;
     EnElforg* this = THIS;
 
-    Actor_SetScale(&this->actor, 0.01f);
-    this->flags = 0;
+    Actor_SetScale(thisx, 0.01f);
```

Comment:
What's the improvement from using thisx here?

Reply:
Elforg was done back before we decided that, so I only used `thisx` for the one macro below. Things have changed since then, so that's why I used `thisx` in Init.